src/OsmMom.sol

```diff
@@ -25,42 +27,39 @@ interface AuthorityLike {
     function canCall(address src, address dst, bytes4 sig) external view returns (bool);
 }
 
+interface VatLike {
+    function file(bytes32 ilk, bytes32 what, uint256 data) external;
+}
+
+interface AutoLineLike {
+    function remIlk(bytes32 ilk) external;
+}
+
 contract OsmMom {
-    event LogNote(
-        bytes4   indexed  sig,
-        address  indexed  usr,
-        bytes32  indexed  arg1,
-        bytes32  indexed  arg2,
-        bytes             data
-    ) anonymous;
-
-    modifier note {
+    address public owner;
+    address public authority;
+    address public autoLine;
+
+    mapping (bytes32 => address) public osms;
+
+    address public immutable vat;
+
+    event SetOwner(address owner);
```

Comment:
comparing to [clipper-mom](https://github.com/makerdao/clipper-mom/blob/master/src/ClipperMom.sol#L49) I wonder if we might want to consider following the same patter here by emitting also the `oldOwner` and use `indexed` as well for consistency.
```suggestion
    event SetOwner(address indexed oldOwner, address indexed newOwner);
```

Reply:
yeah, I agree.
checked `DSAuth` and it looks cleaner.
going to resolve related comments.